### PR TITLE
Add GPUtil to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge/staged-recipes this recipe is ready for review

File: recipes/gputil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 099e52c65e512cdfa8c8763fca67f5a5c2afb63469602d5dcb4d296b3661efb9
+  - url: https://raw.githubusercontent.com/anderskm/gputil/master/LICENSE.txt
+    sha256: 96393a5bcbaa64a621997a88a51b336e686d5eaa6996aaa830b170e15b198e6b
```
Comment (jakirkham) [inline recipes/gputil/meta.yaml:13]: That's a clever solution! 😄

**Comment by jakirkham** [comment]
Thanks @sodre! 😄

\<shameless plug\>
FWIW we've revitalized [pynvml]( https://github.com/gpuopenanalytics/pynvml ) and have been using this to inspect GPU usage from Python and have even added [a JupyterLab dashboard]( https://github.com/rapidsai/jupyterlab-nvdashboard ) if you are exploring tools in this space 😉
\</shameless plug\>

**Comment by sodre** [comment]
Thanks, I’ll propose that instead to the team!

**Comment by jakirkham** [comment]
Cool. Please let us know how it goes. Feedback on them would be most welcome 🙂

**Pull request merged**